sdk/src/main/java/io/radar/sdk/model/RadarEvent.kt

```diff
@@ -97,50 +102,56 @@ class RadarEvent(
      * The types for events.
      */
     enum class RadarEventType {
+        // These strings should match the values (and order) of the server's event constants.
```

Comment:
@jrtibbetts @jsani-radar iirc, changing the order of these enum values changes the mapping from event type --> underlying int value, which can have downstream consequences (e.g., storing in the database in toolkit). Are we concerned about that?

Reply:
That's a good question, but IIRC, when other new values were added, they weren't tacked onto the end, so that would also have caused a raw-value ordering problem then, too.